CHANGELOG.md

```diff
@@ -1,8 +1,30 @@
-## 0.6.0 (Unreleased)
-## 0.5.0 (April 22, 2020)
+## 0.5.1 (Unreleased)
+
+[Full Changelog](https://github.com/terraform-providers/terraform-provider-mongodbatlas/compare/v0.5.0...HEAD)
+
+**Implemented enhancements:**
```

Comment:
This is a hot fix so no enhancements, please remove this section and the one item in it.